code/modules/projectiles/projectile/special/flense.dm

```diff
@@ -0,0 +1,117 @@
+	icon_state = "iv_drip"
+	can_buckle = 1
+	max_integrity = 40
+	density = FALSE
+	can_buckle = TRUE
+	buckle_prevents_pull = TRUE
+	buckle_lying = FALSE
+	var/rended = FALSE
+
+/obj/structure/flensingnet/user_unbuckle_mob(mob/living/dyingman, mob/living/user)
+	if(user == dyingman)
+		user.visible_message("<span class='warning'>[user] starts wriggling off of [src]!</span>", \
+		"<span class='danger'>You start agonizingly working your way off of [src]...</span>")
+		if(!do_after(user, 300, target = user))
+			user.visible_message("<span class='warning'>[user] slides back down [src]!</span>")
+			user.emote("scream")
+			user.apply_damage(10, BRUTE, BODY_ZONE_CHEST)
+			playsound(user, 'sound/misc/desceration-03.ogg', 50, TRUE)
+			return
+	else
+		user.visible_message("<span class='danger'>[user] starts tenderly lifting [src] off of [dyingman]...</span>", \
+		"<span class='danger'>You start tenderly lifting [src] off of [dyingman]...</span>")
+		if(!do_after(user, 60, target = dyingman))
+			dyingman.visible_message("<span class='warning'>[src] painfully rends [dyingman].</span>")
+			if(dyingman.stat >= UNCONSCIOUS)
+				return
+			dyingman.say("Oof, ouch owwie!!", forced = "flensing net removal failure")
+			return
+	dyingman.visible_message("<span class='danger'>[dyingman] comes free of [src] with a squelch!</span>", \
+	"<span class='boldannounce'>You come free of [src]!</span>")
+	dyingman.DefaultCombatKnockdown(30)
+	playsound(dyingman, 'sound/misc/desceration-03.ogg', 50, TRUE)
+	unbuckle_mob(dyingman)
+
+/obj/structure/flensingnet/user_buckle_mob()
+		return
+
+/obj/structure/flensingnet/Initialize()
+	. = ..()
+	START_PROCESSING(SSfastprocess, src)
+	var/mob/living/whale = locate() in get_turf(src)
+	if(whale)
+		if(iscyborg(whale))
+			if(!whale.stat)
+				whale.visible_message("<span class='boldwarning'>The flensing net rends [whale]'s chassis, going blunt and useless in the process!</span>", \
+				"<span class='userdanger'>A razor net rips your chassis and bursts into shrapnel in your casing!</span>")
+				whale.adjustBruteLoss(35)
+				whale.Stun(20)
+				addtimer(CALLBACK(src, .proc/take_damage, max_integrity), 1)
+		else
+			whale.visible_message("<span class='boldwarning'>A flensing net ensnares [whale], embedding itself into their skin!</span>", \
+			"<span class='userdanger'>A flensing net ensnares you, its razors digging painfully into your flesh!</span>")
+			whale.emote("scream")
+			playsound(whale, 'sound/effects/splat.ogg', 50, TRUE)
+			playsound(whale, 'sound/misc/desceration-03.ogg', 50, TRUE)
+			whale.adjustBruteLoss(10)
+		mouse_opacity = MOUSE_OPACITY_OPAQUE //So players can interact with the tile it's on to pull them off
+		buckle_mob(whale, TRUE)
+
+/obj/structure/flensingnet/process()
+	if(buckled_mobs && LAZYLEN(buckled_mobs))
+		var/mob/living/spitroast = buckled_mobs[1]
+		spitroast.adjustBruteLoss(0.1)
+		if(spitroast.health < spitroast.maxHealth/1.3)//around 153 hp on a 200hp mob
```

Comment:
Totally balanced. At least remove projectile's damage

Reply:
yes 0.05 brute/sec is very op